job-service-container/src/test/java/com/hpe/caf/services/job/api/JobServiceIT.java

```diff
@@ -419,15 +421,14 @@ public void testDeleteJobsUsingJobIdsList() throws ApiException
 
         final String filter = String.format("id=in=(%s, %s, %s)", jobIds.get(0), jobIds.get(1), jobIds.get(2));
 
-        final long response = jobsApi.deleteJobs(defaultPartitionId, jobCorrelationId, null, null, filter);
-
+        final InlineResponse2001 response = jobsApi.deleteJobs(defaultPartitionId, jobCorrelationId, null, null, filter);
 
         final List<Job> deletedJobs = jobsApi.getJobs(
                 defaultPartitionId, jobCorrelationId,null,null,null,null,null, null, filter);
 
         assertTrue(deletedJobs.isEmpty());
 
-        assertEquals(response, 3);
+        assertEquals((long) response.getJobsDeleted(), 3L);
```

Comment:
Very minor point for the future.
Casting esponse.getJobsDeleted() to Long can cause an exception if it's null, making it slightly harder to understand why the test failed, if you cast 3L to a Long you can avoid this.

assertEquals(response.getJobsDeleted(), (Long)3L);

No need to change here.